internal/backend/projects/projects.go

```diff
@@ -50,6 +51,14 @@ func (ps *Projects) Create(ctx context.Context, project sdktypes.Project) (sdkty
 			return err
 		}
 
+		// create default cron connection
+		if cronConnection, err := tx.GetConnection(ctx, fixtures.BuiltinSchedulerConnectionID); errors.Is(err, sdkerrors.ErrNotFound) {
```

Comment:
interesting. not sure it's needed? if it's built in, no need to display it i think.

Reply:
it should be in the DB. maybe we could add additional column that will mark bultin connections, but right now I think this is the simplest way. Do you thought about a better way?